external-crates/move/crates/move-disassembler/src/disassembler.rs

```diff
@@ -444,6 +445,14 @@ impl<'a> Disassembler<'a> {
         }
     }
 
+    fn preview_string(s: &str) -> String {
+        if s.len() <= 4 {
```

Comment:
This is a pet peeve of mine 😅 , but this bound should be `6` to make sure the preview is always smaller than the input string.

Reply:
Indeed! Very good spot. Will update :)